src/olympia/addons/tests/test_models.py

```diff
@@ -1430,11 +1430,12 @@ def test_listed_is_incomplete_no_license(self):
     def test_unlisted_is_incomplete(self):
         addon = Addon.objects.get(id=3615)
         addon.update(is_listed=False)
-        addon.current_version.update(channel=amo.RELEASE_CHANNEL_UNLISTED)
+        addon.versions.update(channel=amo.RELEASE_CHANNEL_UNLISTED)
         assert not addon.is_incomplete()  # Confirm not incomplete already.
 
         # Clear everything
-        addon.current_version.update(license=None)
+        addon.find_latest_version(
+            channel=amo.RELEASE_CHANNEL_UNLISTED).update(license=None)
```

Comment:
Yeah. I should just do `addon.versions.update(license=None)` really, no need to filter, it's just to set up some test data